<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contentWindowClass</class>
 <widget class="QMainWindow" name="contentWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1577</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>content</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1561</width>
      <height>841</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Constant Curvature Model</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>630</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>视角左移</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>961</width>
        <height>541</height>
       </rect>
      </property>
      <property name="text">
       <string>frameflow</string>
      </property>
     </widget>
     <widget class="QPushButton" name="down_pushbutton_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>660</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>视角下移</string>
      </property>
     </widget>
     <widget class="QPushButton" name="up_pushbutton_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>600</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>视角上移</string>
      </property>
     </widget>
     <widget class="QPushButton" name="right_pushbutton_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>630</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>视角右移</string>
      </property>
     </widget>
     <widget class="QPushButton" name="is_rotation_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>630</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>旋转</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser1">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>570</y>
        <width>561</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_log">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>760</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清除记录</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>V2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>V3</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>V4</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>V5</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1577</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
